content/300-guides/200-deployment/201-serverless/500-deploy-to-netlify.mdx

```diff
@@ -237,44 +67,45 @@ You can now create your Netlify site and configure continuous deployments for it
+   5. _Your build command_: The command you use to build your project. Common examples include `npm run build`, `yarn run build`, `next build`, etc.
+   6. _Directory to deploy_: Generally, the directory that your `build` command outputs to. Common examples include `dist/`, `build/`, `.next`, etc.
+   7. Allow Netlify to create a `netlify.toml` file.
 
-### Result
+Netlify has now setup your site and connected your github repo to it! You should now be able to use `netlify open` to view the Admin page of your newly created Site.
 
-Check that the Netlify CLI shows the following:
+You can open your app now, but you may see errors when you use options that connect to the database. This is because Prisma will look to connect to your database, but if your connection string is setup as an environment variable [Netlify will not recognize it by default](https://docs.netlify.com/environment-variables/get-started/#work-with-env-files).
 
-```text
-Site Created
+## 8. Store environment variables in Netlify
 
-Admin URL: https://app.netlify.com/sites/YOUR_SITE_NAME
-URL:       https://YOUR_SITE_NAME.netlify.app
-```
+We recommend keeping `.env` files in your `.gitignore` in order to prevent leakage of sensitives connection strings. Instead, you can use the Netlify CLI to [import values into netlify directly](https://docs.netlify.com/environment-variables/get-started/#import-variables-with-the-netlify-cli).
 
-> **Note:** You can change the build configuration in the `netlify.toml` file and the site name from the Netlify admin UI.
+Assuming you have a file like the following:
 
-Netlify deploys your app for the first time to the designated URL. Also, your application is now configured for continuous deployments. Every time you make a push to the GitHub repository, Netlify starts a new build of the site.
+```env file=.env
+# Connect to DB
+DATABASE_URL="postgresql://postgres:__PASSWORD__@__HOST__:__PORT__/__DB_NAME__"
+```
 
-You can call the API status endpoint (which is part of our example application) with `curl` to verify that your application is deployed.
+You can upload the file as environment variables using the `env:import` command
 
 ```terminal
-curl https://YOUR_SITE_NAME.netlify.app/.netlify/functions/status
+❯ netlify env:import .env
+site: my-very-very-cool-site
+.---------------------------------------------------------------------------------.
+|                         Imported environment variables                          |
+|---------------------------------------------------------------------------------|
+|     Key      |                              Value                               |
+|--------------|------------------------------------------------------------------|
+| DATABASE_URL | postgresql://postgres:__PASSWORD__@__HOST__:__PORT__/__DB_NAME__ |
```

Comment:
This is rendered terribly: 
<img width="593" alt="image" src="https://github.com/prisma/docs/assets/183673/a0f508c7-486c-4085-99d5-c85cb86b2431">

I think `|` is used to trigger some highlighting...

Can this be disabled somehow @nilubava @carlagn?

Reply:
:ohno: